US/CA/san-francisco.md

```diff
@@ -0,0 +1,13 @@
+## San Francisco, CA
+  - Places to eat:
```

Comment:
Which section of SF are these in? I will be there next week and would :heart: some suggestions for new places to try!